src/helpers/elements-interactions.js

```diff
@@ -11,8 +11,7 @@ async function fillInput(page, inputSelector, inputValue) {
 }
 
 async function clickButton(page, buttonSelector) {
-  const button = await page.$(buttonSelector);
-  await button.click();
+  await page.$eval(buttonSelector, (el) => el.click());
```

Comment:
I don't know why only for me the `page.$(selector).click()` don't work, but the `eval` way did.

(I think I was the one that changed this in the `link` click)

Reply:
This method is used in other companies so to change it we now need to test them all. 
Maybe it is better that you will copy this method into `mizrahi` scraper and use it locally. That way this PR will not affect other scrapers beside Mizrahi and we will be able to merge it. wdyt?